### ISSUE TITLE
Crash when trying to switch to my currently playing playlist

### ISSUE THREAD

**Issue by AnotherLife** [TeamNewPipe/NewPipe#3320]
This is a super-frequent crash that happens when I am playing a video on popup (that I have started from the What's New tab) and I tap on the Notification in order to get to the playlist I am currently watching. I don't mean a normal playlist, I mean the videos I have been enqeuing and watching.

```
{
   "user_action": "ui error",
   "exceptions": [
      "android.content.res.Resources$NotFoundException: File res\/drawable\/ic_volume_up_white_24dp.xml from drawable resource ID #0x7f08015c. If the resource you are trying to use is a vector resource, you may be referencing it in an unsupported way. See AppCompatDelegate.setCompatVectorFromResourcesEnabled() for more info.\n\tat android.content.res.Resources.loadDrawable(Resources.java:2103)\n\tat android.content.res.Resources.getDrawable(Resources.java:700)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.getThemedDrawable(ServicePlayerActivity.java:706)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.onMaybeMuteChanged(ServicePlayerActivity.java:701)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.onCreateOptionsMenu(ServicePlayerActivity.java:156)\n\tat android.app.Activity.onCreatePanelMenu(Activity.java:2538)\n\tat androidx.fragment.app.FragmentActivity.onCreatePanelMenu(FragmentActivity.java:325)\n\tat androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:94)\n\tat androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.onCreatePanelMenu(AppCompatDelegateImpl.java:2830)\n\tat androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:94)\n\tat androidx.appcompat.app.ToolbarActionBar.populateOptionsMenu(ToolbarActionBar.java:455)\n\tat androidx.appcompat.app.ToolbarActionBar$1.run(ToolbarActionBar.java:56)\n\tat android.os.Handler.handleCallback(Handler.java:733)\n\tat android.os.Handler.dispatchMessage(Handler.java:95)\n\tat android.os.Looper.loop(Looper.java:136)\n\tat android.app.ActivityThread.main(ActivityThread.java:5133)\n\tat java.lang.reflect.Method.invokeNative(Native Method)\n\tat java.lang.reflect.Method.invoke(Method.java:515)\n\tat com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)\n\tat com.android.internal.os.ZygoteInit.main(ZygoteInit.java:602)\n\tat dalvik.system.NativeStart.main(Native Method)\nCaused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector\n\tat android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:933)\n\tat android.graphics.drawable.Drawable.createFromXml(Drawable.java:877)\n\tat android.content.res.Resources.loadDrawable(Resources.java:2099)\n\t... 20 more\norg.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector\n\tat android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:933)\n\tat android.graphics.drawable.Drawable.createFromXml(Drawable.java:877)\n\tat android.content.res.Resources.loadDrawable(Resources.java:2099)\n\tat android.content.res.Resources.getDrawable(Resources.java:700)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.getThemedDrawable(ServicePlayerActivity.java:706)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.onMaybeMuteChanged(ServicePlayerActivity.java:701)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.onCreateOptionsMenu(ServicePlayerActivity.java:156)\n\tat android.app.Activity.onCreatePanelMenu(Activity.java:2538)\n\tat androidx.fragment.app.FragmentActivity.onCreatePanelMenu(FragmentActivity.java:325)\n\tat androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:94)\n\tat androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.onCreatePanelMenu(AppCompatDelegateImpl.java:2830)\n\tat androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:94)\n\tat androidx.appcompat.app.ToolbarActionBar.populateOptionsMenu(ToolbarActionBar.java:455)\n\tat androidx.appcompat.app.ToolbarActionBar$1.run(ToolbarActionBar.java:56)\n\tat android.os.Handler.handleCallback(Handler.java:733)\n\tat android.os.Handler.dispatchMessage(Handler.java:95)\n\tat android.os.Looper.loop(Looper.java:136)\n\tat android.app.ActivityThread.main(ActivityThread.java:5133)\n\tat java.lang.reflect.Method.invokeNative(Native Method)\n\tat java.lang.reflect.Method.invoke(Method.java:515)\n\tat com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)\n\tat com.android.internal.os.ZygoteInit.main(ZygoteInit.java:602)\n\tat dalvik.system.NativeStart.main(Native Method)\n"
   ],
   "os": "Linux Android 4.4.4 - 19",
   "time": "2020-03-31 22:33",
   "request": "App crash, UI failure",
   "package": "org.schabi.newpipe",
   "service": "none",
   "content_language": "GB",
   "user_comment": "",
   "version": "0.19.1"
}
```

What causes this error? And why are there backslashes and forward slashes there in that filename?

**Comment by mauriciocolli** [comment]
@TobiGr @B0pol @Stypox  [:man_facepalming: ](https://www.youtube.com/watch?v=3O8J2locx5o)

https://github.com/TeamNewPipe/NewPipe/blob/f7697007e5577bf4ccf24a7b1b04445cc17d1725/app/src/main/java/org/schabi/newpipe/player/ServicePlayerActivity.java#L700-L709

---

@AnotherLife 
> This is a super-frequent crash [...]

Yes, in fact, super-frequent is an understatement, it's impossible to open that screen in your version.

> And why are there backslashes and forward slashes there in that filename?

The library that is used to build the JSON string opted to escape the forward slash, although it's just optional. I think the standard one from Android is being used right now.

**PS**: Always try to follow the [issue template](https://github.com/TeamNewPipe/NewPipe/issues/new?template=bug_report.md), it's easier for us to see exactly what the issue is (for example, you didn't used the ["incredible bugreport to markdown converter"](https://teamnewpipe.github.io/CrashReportToMarkdown/) mentioned there).

**Comment by Stypox** [comment]
@mauriciocolli I have already fixed this in #3271, if needed I can open another pr (will do in 1 hour if nobody answers)

**Comment by Stypox** [comment]
By the way, if I remember correctly that `getThemedDrawable` was remved because it was a duplicate of `ThemeHelper.resolveResourceIdFromAttr`, but apparently it reappeared :thinking:
